packages/app/tsconfig.json

```diff
@@ -0,0 +1,62 @@
+ 
+{
+  "compilerOptions": {
```

Comment:
"include": ["src"],
  "extends": "../../tsconfig.base.json",
  "compilerOptions": {
    "target": "es6",
    "allowJs": true,
  }

add this

Reply:
We need to discuss how we are going to handle the versions of ts, native, etc. Actually the project are looking for reference at their root folder, but using a monorepo architecture will set-up all dependencies at the root folder.